### DISCUSSION TITLE
Problem with mint in-game currency

### DISCUSSION THREAD

**Discussion by netluke** [ton-org/game-engines-sdk#13] [Q&A]
Hi Ali, my name is Luca and I want to develop a game on Telegram.

I'm following this guide:
https://docs.ton.org/develop/dapps/tutorials/building-web3-game
But after finishing the "Mint in-game currency" part, I get this result:

![mint_problem](https://github.com/user-attachments/assets/fa43a85d-b52a-4ab6-ab8d-9a4eabf817ff)
Instead of the one reported in the guide:

![mint_good](https://github.com/user-attachments/assets/ced5d7ea-6678-471e-a6cb-d7afc13c3cf5)

I thought the problem was loading the image, because I gave a path on Pinata, and than I have created several Jettons by changing the image path, but the result is always the same.

**I'm stuck, can you help me?**

**Comment by barinbritva** [comment]
@netluke hi,

Sure, I will help you to address the issue.

Could you please provide some details?
1. Which version of `@ton-community/assets-sdk` / `assets-cli` do you have installed
2. Please attach step-by-step screenshots of `assets-cli deploy-jetton` process

After that I will be able to reproduce the process and help find the issue

**Reply by netluke** [reply]
Hi, thanks for the reply, here is the information you were asking for:

Versions:
+-- @ton-community/assets-sdk@0.0.4
+-- @ton-community/gamefi-sdk@0.0.3
+-- boxen@5.1.2
+-- chalk@4.1.2
`-- npm@10.8.2

All packages are installed as "global".

Deploy Jetton process:
C:\Users\nadia>assets-cli deploy-jetton
? Enter jetton name Flappy Jetton
? Enter jetton description Flappy Bird Token.
? Enter image path or link C:\temp\FlappyJetton.png
? Enter jetton symbol (for example TON) FBT
? Enter jetton decimals (for example 9) 0

╭ Flappy Jetton ─────────────────────────────────────────────────────────────────╮
│ description: Flappy Bird Token.                                                                                                                                                │
│ image: ipfs://QmRtgnxkMtvBbBX2ZdAVvfRzHxQXTn7sNhkTKaZrrCpfq8                                                                                │
│ symbol: FBT                                                                                                                                                                              │
│ decimals: 0                                                                                                                                                                               │
│ jetton address: kQDum29iiLqdT4ncAAzOaUhEem8oMpxi2f8XMzrh39-rb0co                                                                        │
│ https://testnet.tonviewer.com/kQDum29iiLqdT4ncAAzOaUhEem8oMpxi2f8XMzrh39-rb0co                                                │
╰──────────────────────────────────────────────────────────────────────────╯

I hope this informations are useful for you.
Thanks
